Dockerfile

```diff
@@ -1,11 +1,14 @@
 FROM ubuntu:18.04
 
+COPY . style-analyzer
```

Comment:
Here is the problem: as soon as you change anything inside, it rebuilds all the layers. Better:

```
COPY requirements.txt style-analyzer

RUN ...

COPY . style-analyzer
```

This way it rebuilds the world only if there are different deps, which is rare.